pages/common/qtcreator.md

```diff
@@ -0,0 +1,28 @@
+# qtcreator
+
+> Cross-platform IDE for Qt applications.
+> More information: <https://doc.qt.io/qtcreator/creator-cli.html>.
+
+- Start Qt Creator:
+
+`qtcreator`
+
+- Start Qt Creator and restore the last session:
+
+`qtcreator -lastsession`
+
+- Start Qt Creator but don't load the specified plugin:
+
+`qtcreator -noload {{plugin}}`
+
+- Start Qt Creator but don't load any plugins:
+
+`qtcreator -noload {{all}}`
+
+- Start Qt Creator in presentation mode with pop-ups for key combos:
+
+`qtcreator -presentationMode`
+
+- Start Qt Creator and show a specific commit hash:
+
+`qtcreator -git-show {{commit_hash}}`
```

Comment:
This description is a bit confusing.  "show a specific commit hash" doesn't necessarily mean anything to infrequent users of the command 🤔

Reply:
I just tested it and it seems to show the diff between the specified commit and the previous one. I don't really know how to correctly say that in "git" language tough.

Reply:
I think we use `diff` to say that ([git-show.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/git-show.md)):
```
- Start Qt Creator and show the diff from a specific commit:

`qtcreator -git-show {{commit}}`
```